src/sidebar/components/test/group-list-test.js

```diff
@@ -85,11 +75,7 @@ describe('groupList', function() {
 
     fakeAnalytics = {
       track: sinon.stub(),
-      events: {
-        GROUP_LEAVE: 'groupLeave',
-        GROUP_SWITCH: 'groupSwitch',
-        GROUP_VIEW_ACTIVITY: 'groupViewActivity',
-      },
+      events: events,
```

Comment:
This is fine, but idiomatically if the key and value are the same it can be shortened to just the key name (ie. `fakeAnalytics = { track: sinon.stub(), events }`